sites/postcss-preset-env/.eleventy.js

```diff
@@ -14,6 +15,15 @@ module.exports = function (eleventyConfig) {
 
 	eleventyConfig.addPlugin(syntaxHighlight);
 
+	const now = new Date();
```

Comment:
unused?

Reply:
Yeah I was going to filter but decided not to. Good catch!